packages/blade/src/components/Alert/__tests__/Alert.web.test.tsx

```diff
@@ -96,31 +96,14 @@ describe('<Alert />', () => {
     const { getByRole } = renderWithTheme(
       <Alert
         description="Currently you can only accept payments in international currencies using PayPal."
-        intent="positive"
+        color="positive"
       />,
     );
     const alert = getByRole('status');
 
     await assertAccessible(alert);
   });
 
-  it('should throw an error if secondary action is defined without primary action', () => {
```

Comment:
Why did we remove this test?

Reply:
We've removed this restriction.
<img width="815" alt="image" src="https://github.com/razorpay/blade/assets/35374649/487d5633-34a5-47bf-b2e8-903c3811f0dd">